discord_thread_summarizer_using_openai/main.bal

```diff
@@ -0,0 +1,75 @@
+import ballerina/io;
+import ballerina/log;
+import ballerina/time;
+
+const TYPE_FORUM_CHANEL = 15;
+const time:Seconds ONE_DAY_IN_SECONDS = 86400;
+configurable int forumChannelId = ?;
+
+function readThreads(ActiveThreads activeThreads) returns error? {
+
+    foreach ChannelThread thread in activeThreads.threads {
+
+        // check if the thread has any recent messages
+        if !check hasRecentMessages(thread) {
+            continue;
+        }
+
+        string prompt = check constructPrompt(thread);
+        io:println(prompt);
```

Comment:
Shall we revisit the `io:println` statements? Do we really need all of these? If we do, shouldn't we use something like

```suggestion
        io:println("Prompt: ", prompt);
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/9018e16abefb68f7d46cfbc2a86a00240ce6ea5f